changelogs/v0.3.0.md

```diff
@@ -0,0 +1,18 @@
+# Changes
+
+* [FEATURE] Added loki backend support for the rules commands, configurable with `--backend=loki` (defaults to cortex).
+* [FEATURE] Introduces a new `version` command. The command will also let you know if there's a new version available.
```

Comment:
should this be in the root `CHANGELOG.md` as well?

Reply:
Addressed in #86 can I get a review?